app.js

```diff
@@ -0,0 +1,51 @@
+/* eslint no-use-before-define:["error",{"functions":false}] */
+let library = [];
+
+// eslint-disable-next-line no-unused-vars
+function addBook() {
+  const book = {};
+  book.title = document.getElementById('Title').value;
+  book.author = document.getElementById('Author').value;
+  library.push(book);
+  displayBooks();
+  saveLibrary();
+}
+
+function removeBook(title) {
+  library = library.filter((book) => book.title !== title);
+  displayBooks();
+  saveLibrary();
+}
+
+function displayBooks() {
+  const libraryDiv = document.getElementById('library');
+  libraryDiv.innerHTML = '';
+  library.map((book) => {
+    const divBook = document.createElement('div');
+    const p = document.createElement('p');
+    p.innerHTML = book.author;
```

Comment:
[Optional]
If you're just changing the inner text, `textContent` would be more appropriate. (And it's safer than `innerHTML`) 
```suggestion
    p.textContent = book.author;
```

Reply:
Ok, thanks.